### PR TITLE
Swift: extract availability info

### PR DESCRIPTION
**Pull request by AlexDenisov**

### CODE REVIEW

File: swift/schema.py [hunk 1]
```diff
@@ -811,11 +811,47 @@ class CaseLabelItem(AstNode):
...
+      // Runs on iOS 12 and above
+    } else {
+      // Runs only anything below iOS 12
+    }
+    if #unavailable(macOS 10.14, *) {
+      // Runs only on macOS 10 and below
+    }
+    ```
+    """
+    unavailability: boolean
```
Comment (MathiasVP) [inline swift/schema.py:846]: Could this maybe be specified as a `: predicate` instead? I think this would read more naturally in QL then.

**Review by MathiasVP — APPROVED** [review]
The changes LGTM! Should we run a DCA on it?

**Comment by AlexDenisov** [comment]
> The changes LGTM! Should we run a DCA on it?

hmm, what can we learn from it? 🤔

**Comment by AlexDenisov** [comment]
> > hmm, what can we learn from it? 🤔
> 
> That there's not some null pointer dereference that crashes the extractor somewhere? 😄

Yeah, fair 😄
I'll start one.

**Comment by MathiasVP** [comment]
DCA looks fine!

**Pull request merged**